package.json

```diff
@@ -157,9 +157,40 @@
     "packages/*"
   ],
   "jest": {
+    "collectCoverage": true,
+    "coverageProvider": "v8",
+    "testPathIgnorePatterns": [
+      "<rootDir>/packages/spec/.tmp/"
+    ],
+    "watchPlugins": [
+      "jest-runner-eslint/watch-fix",
+      "jest-watch-select-projects",
+      "jest-watch-typeahead/filename",
+      "jest-watch-typeahead/testname"
+    ],
     "projects": [
       "<rootDir>/packages/!(spec)",
-      "<rootDir>/packages/spec/integration/*"
+      "<rootDir>/packages/spec/integration/*",
```

Comment:
- renaming `test` to `unit` is fine for me
- renaming `template` to `unit` is fine for me
- I'm not sure though, how we can provide the config on the top-level, because through the glob expression each one of them becomes a "project" entry. So in order to provide them here, we'd need to add a "project" entry for each package, I guess? Or have you found a different way?

Reply:
Oh, didn't know of that implication of the glob expression here. It's a little unfortunate to have that project name in the templates's `package.json` files. But I don't have a better idea…

Reply:
I've renamed `test` and `templates` to `unit`